cpp/fetcher/continuous_fetcher.cc

```diff
@@ -30,7 +31,8 @@ namespace {
 class ContinuousFetcherImpl : public ContinuousFetcher {
  public:
   ContinuousFetcherImpl(libevent::Base* base, Executor* executor,
-                        Database<LoggedCertificate>* db, bool fetch_scts);
+                        Database<LoggedCertificate>* db,
+                        const LogVerifier* log_verifier, bool fetch_scts);
```

Comment:
Based on it not being a `unique_ptr<const LogVerifier>`, I'd say the caller keeps ownership. :smiley: